index.html

```diff
@@ -53,6 +53,168 @@ <h1>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card_image">
+                    <img src="logo/Img Placeholder.png" alt="Image placeholder">
+                </div>
+                <div class="card_content">
+                    <h3>Multi-Post Stories <br> Gain+Glory</h3>
+                    <ul>
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card_image">
+                    <img src="logo/Img Placeholder.png" alt="Image placeholder">
+                </div>
+                <div class="card_content">
+                    <h3>Multi-Post Stories <br> Gain+Glory</h3>
+                    <ul>
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card_image">
+                    <img src="logo/Img Placeholder.png" alt="Image placeholder">
+                </div>
+                <div class="card_content">
+                    <h3>Multi-Post Stories <br> Gain+Glory</h3>
+                    <ul>
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card_image">
+                    <img src="logo/Img Placeholder.png" alt="Image placeholder">
+                </div>
+                <div class="card_content">
+                    <h3>Multi-Post Stories <br> Gain+Glory</h3>
+                    <ul>
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- About -->
+    <section class="about">
+        <div class="about_content">
+            <div>
+                <div class="illustration1">
+                    <h3>About Me</h3>
+                    <img src="logo/illustration 1- about me-mobile.png" alt="">
+                </div>
```

Comment:
- [ ] As stated in the project requirement the website should be responsive, following these breakpoints:Minimum mobile screen size should be 375px up to 768px But in the case of your About me heading is not responsive So kindly Fix it.
- Your About me heading start from 387-768 PX mobile screen size
 
![About me](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/93598189/3fed4e69-b09b-4f84-92d3-f3c7c2f964e2)